memory/libhugetlbfs.py

```diff
@@ -99,7 +99,7 @@ def setUp(self):
                 self.hugetlbfs_dir = os.path.join(self.tmpdir, 'hugetlbfs')
                 os.makedirs(self.hugetlbfs_dir)
             process.system('mount -t hugetlbfs none %s' %
-                           self.hugetlbfs_dir, sudo=True)
+                           self.hugetlbfs_dir, sudo=True, ignore_status=True)
```

Comment:
Test should not continue after this. We may need to cancel if this fails.

Reply:
@harish-24 ah yes will take care